src/llv8.h

```diff
@@ -20,28 +21,33 @@ class CodeMap;
 
 class Error {
  public:
-  Error() : failed_(false), msg_(nullptr) {}
-  Error(bool failed, const char* msg) : failed_(failed), msg_(msg) {}
+  Error() : failed_(false), msg_("") {}
+  Error(bool failed, std::string msg) : failed_(failed), msg_(msg) {}
+  Error(bool failed, const char* format, ...);
```

Comment:
Consider tagging the variadic functions with `__attribute__((format(printf, 2, 3)))` to have the compiler check the format string (`__attribute__((format(printf, 1, 2)))` for the ones below.)